packages/workspace/src/utilities/perf-logging.ts

```diff
@@ -5,5 +5,5 @@ if (process.env.NX_PERF_LOGGING) {
     const entry = list.getEntries()[0];
     console.log(`Time for '${entry.name}'`, entry.duration);
   });
-  obs.observe({ entryTypes: ['measure'], buffered: false });
```

Comment:
Edited in correlation to details here: https://developer.mozilla.org/en-US/docs/Web/API/PerformanceObserver/observe#syntax

`buffered` only required if `type` is used